server/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/temporalio/ui-server/v2
 
-go 1.20
+go 1.21
+
+toolchain go1.23.0
```

Comment:
Do we want to add `toolchain go1.23.0` here?

Reply:
👍